### PR TITLE
ADD: CoinControl multiselect

### PR DESCRIPTION
**Pull request by limpbrains**
- show number of confirmations in Output Modal
- allow to select a few outputs and Freeze/Unfreeze/Use them

### CODE REVIEW

**Review by Overtorment — APPROVED** [review]
lgtm

File: screen/send/coinControl.js [hunk 1]
```diff
@@ -35,87 +39,157 @@ const debounce = (func, wait) => {
...
+    freezeDark: { backgroundColor: colors.redBG, borderWidth: 0 },
+    freezeText: { color: colors.redText },
+  });
+
+  return <Badge value={loc.cc.freeze} badgeStyle={oStyles[cs === 'dark' ? 'freezeDark' : 'freezeLight']} textStyle={oStyles.freezeText} />;
+};
+
+const ChangeBadge = () => {
+  const { colors } = useTheme();
+  const cs = useColorScheme();
```
Comment (marcosrdz) [inline screen/send/coinControl.js:56]: useColorScheme? You can access colors directly and it will select the correct one based on the theme.

**Review by marcosrdz — CHANGES_REQUESTED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/u3xdjbw2dcuzuccc6f6fn5b5p0

[download apk](http://filestorage.bluewallet.io:1488/1607796957.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/865dqptmaf5dc9mgpmq1w6hu5r

[download apk](http://filestorage.bluewallet.io:1488/1607846532.apk)

**Review by marcosrdz — APPROVED** [review]

**Pull request merged**